.github/workflows/test-integration-v2-general-auth.yml

```diff
@@ -0,0 +1,35 @@
+name: Integration Test Authentication v2
```

Comment:
Lets not split these anymore, the new tests can run in parallell, so lets not start by splitting them.